<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Intersection/Intersection_icon.png</normaloff>:/Intersection/Intersection_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_Main">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>31</y>
      <width>251</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white;
border-radius:5px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:skyblue;
color:white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>실시간 교차로
시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>390</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:skyblue;
color:white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>기존 교차로
시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>650</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f05650;
color:white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_South_text">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>SOUTH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_West">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_North">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_South">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetime">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>바탕</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:#D99C52;
</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_West_text">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>W
E
S
T</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_East_text">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>E
A
S
T</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_North_text">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>NORTH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_East">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="Simulation_background">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>825</width>
      <height>725</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/Intersection/intersection.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_West_trafficlight">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>61</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
background:url(:/TrafficLight/TrafficLight/West/West_TrafficLight_Yellow.png);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_East_trafficlight">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>61</width>
      <height>138</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
background:url(:/TrafficLight/TrafficLight/East/East_TrafficLight_Yellow.png);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_South_trafficlight">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>138</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
background:url(:/TrafficLight/TrafficLight/South/South_TrafficLight_Yellow.png);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_North_trafficlight">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>138</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
background:url(:/TrafficLight/TrafficLight/North/North_TrafficLight_Yellow.png);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="page_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>540</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(100, 200,100);
color:white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>이전 페이지</string>
    </property>
   </widget>
   <zorder>label_North_text</zorder>
   <zorder>label_East_text</zorder>
   <zorder>label_West_text</zorder>
   <zorder>label_South_text</zorder>
   <zorder>Simulation_background</zorder>
   <zorder>textBrowser_Main</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>lcdNumber_West</zorder>
   <zorder>lcdNumber_North</zorder>
   <zorder>lcdNumber_South</zorder>
   <zorder>label_datetime</zorder>
   <zorder>lcdNumber_East</zorder>
   <zorder>label_West_trafficlight</zorder>
   <zorder>label_East_trafficlight</zorder>
   <zorder>label_South_trafficlight</zorder>
   <zorder>label_North_trafficlight</zorder>
   <zorder>page_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
